lib/ItemRow/stories/ItemRowStory.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, object, text } from '@storybook/addon-knobs';
+import { ItemRow, StatusIndicator } from '../../index';
+import notes from '../README.md';
+import Icon from '../../Icon';
+import { AvatarGroupMock } from '../../Avatar/stories/AvatarGroupMock';
+
+const createStatusIndicator = props => <StatusIndicator {...props} />;
+
+storiesOf('Components', module).add(
+  'Item Row',
+  () => {
+    const stacked = boolean('Stacked', false);
+    const bordered = boolean('Bordered', true);
+    const commentCount = text('Comment count', 0);
+    const templateName = text('Template name', '');
+
+    return (
+      <ItemRow bordered={bordered} stacked={stacked}>
+        <ItemRow.Name>
+          {!stacked &&
```

Comment:
What's the `stacked` value?

Reply:
It's set in knobs above, it's a boolean and adds a modifier.

The problem is that it requires a little more than just the CSS modifier to be valid, it requires a slight composition re-jig on the component side.

Hense this code.

Reply:
So what does it do in human terms? eg. "You would turn the stacked modifier on if you wanted to see..."?

Reply:
You'd turn it on if you wanted the elements to stack so;

- name
- aside
-- data
-- data

By default they are inline 😄

Reply:
If you open storybook up you can see it visually, it's pretty obvious.